.github/workflows/workflow.yaml

```diff
@@ -34,11 +34,13 @@ jobs:
           pip install -U setuptools pip
           pip install pytest pytest-cov codecov pytest-xdist
           python setup.py install
+      - name: fetch podman image
+        run: sudo podman pull quay.io/redhatqe/selenium-standalone:latest
```

Comment:
This one is not necessary.

Reply:
i consider it a nice add as it takes the fetching out of the test run (